components/src/dynamo/vllm/handlers.py

```
@@ -72,6 +72,66 @@
+            logger.debug(
+                f"Deferred abort: first token not received for request "
+                f"{self._request_id}, spawning background task"
+            )
+            asyncio.create_task(self._wait_and_abort())
+
+    async def _wait_and_abort(self) -> None:
+        """Background task: wait for first token, then abort."""
+        try:
+            await self._first_token_event.wait()
+        except Exception:
+            pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)